src/App.jsx

```diff
@@ -1,3 +1,239 @@
-export const App = () => {
-  return <div>Find me in src/app.jsx!</div>;
-};
+// import React, { useState, useEffect } from 'react';
+// import './index.css';
+
+// const HelloWorld = () => {
+//   return (
+//     <p>HelloWorld!</p>
+//   );
+// };
+
+// export const App = () => {
+//   const [invisible, setInvisible] = useState(false);
+//   const [name, setName] = useState("Jennie");
+
+//   useEffect(() => {
+//     console.log("App effect", invisible);
+//   }, [invisible]);
+
+//   // Function to toggle between "Jennie" and "Bob"
+//   const toggleName = () => {
+//     if (name === "Jennie") {
+//       setName("Bob");
+//     } else {
+//       setName("Jennie");
+//     }
+//   };
+
+//   return (
+//     <React.Fragment>
+//       <button onClick={() => setInvisible(prev => !prev)}>Show / Hide</button>
+//       <button onClick={toggleName}>Change Name</button>
+//       {invisible && <HelloWorld />}
+
+//       <h1>The textarea element</h1>
+
+//       <form action="/action_page.php">
+//         <p><label htmlFor="w3review">Review of W3Schools:</label></p>
+//         <textarea id="w3review" name="w3review" rows="4" cols="50">At w3schools.com you will learn how to make a website. They offer free tutorials in all web development technologies.</textarea>
+//         <br />
+//         <input type="submit" value="Submit" />
+//       </form>
+
+//       <p>Click the &quot;Submit&quot; button and the form-data will be sent to a page on the server called  &quot;action_page.php &quot;.</p>
+//       <p>Name: {name}</p>
+//     </React.Fragment>
+//   );
+// };
+
+import React, { useState, useEffect } from 'react';
+import './index.css'; // Import your CSS file
+import TimeAgo from './components/TimeAgo'; // Import the TimeAgo component
+import {Footer} from './components/footer';
+
+
+async function fetchThoughts() {
+  try {
+    const response = await fetch('https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts');
+    if (!response.ok) {
+      throw new Error('Network response was not ok');
+    }
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    console.error('Error fetching thoughts:', error);
+  }
+}
+
+async function submitThought(message) {
+  try {
+    console.log('Submitting thought:', message);
+    const response = await fetch('https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ message }),
+    });
+    console.log('API response:', response);
+    if (!response.ok) {
+      throw new Error('Network response was not ok');
+    }
+    // Handle successful submission
+    // You may want to refresh the thoughts list or clear the input field
+  } catch (error) {
+    console.error('Error submitting thought:', error);
+  }
+}
+
+export function App() {
```

Comment:
Maybe this export should be in the top of the component and not between all the other functions - makes it a bit hard to find